widget/charts/src/components/BarChart/BarChart.helpers.ts

```diff
@@ -0,0 +1,76 @@
+import type { BarStackDataType } from './BarChart.types';
+
+import { buildChartTheme } from '@visx/xychart';
+
+export const transactionTheme = buildChartTheme({
```

Comment:
We do not need `transactionTheme` and `volumeTheme`.

Reply:
You're right. fixed